packages/dds/sequence/src/sequence.ts

```diff
@@ -459,9 +458,9 @@ export abstract class SharedSegmentSequence<T extends ISegment>
 		// local ops until loading is complete, and then
 		// they will be present
 		if (!this.loadedDeferred.isCompleted) {
-			this.loadedDeferredOutgoingOps.push(metadata ? [translated, metadata] : translated);
+			this.loadedDeferredOutgoingOps.push(metadata ? [message, metadata] : (message as any));
```

Comment:
Side note - The code that reads from this `loadedDeferredOutgoingOps` doesn't seem to account for the case where `metadata === undefined`...

@anthony-murphy - do you see what I'm talking about? AFAIK, the destructuring on line 873 (the `for` line) would throw in this case.

Reply:
loadedDeferredOutgoingOps is unused. it should be removed

Reply:
kind of. there is a flag that enables returning after loading the header, its not used, only in that case should loadedDeferredOutgoingOps have any ops in it.